### ISSUE TITLE
macOS Mojave Screenshot Compatibility Issue

### ISSUE THREAD

**Issue by technolingo** [asweigart/pyscreeze#39]
As title, the screenshot functionality does not work on macOS Mojave, as detailed on this thread:

https://github.com/asweigart/pyautogui/issues/269

**Comment by asweigart** [comment]
Fixed in d554797ce567e65697119c1f229a7de6dc6586c3

**Comment by dgentry** [comment]
I was about to report this, but it looks like it's been fixed.  How do I get the fix, though?  `pip install pyautogui` gets me a version that still has the error.

```
$ pip install pyautogui
Requirement already satisfied: pyautogui in /Users/dennis/.virtualenv/3/lib/python3.7/site-packages (0.9.39)
. . .
Requirement already satisfied: pyscreeze in /Users/dennis/.virtualenv/3/lib/python3.7/site-packages (from pyautogui) (0.1.18)
$
```

Thanks for pyautogui in general!
